BUILD.md

```diff
@@ -71,12 +72,12 @@
   `ssg-${PRODUCT}-guide-index.html`. For example `ssg-rhel7-guide-index.html`.
 
 4. Discover the following:
- * A pretty prose guide **in rhel6-guide.html** containing practical, actionable information for administrators
- * A concise spreadsheet representation (potentially useful as the basis for an SRTM document) in **rhel6-table-nistrefs-server.html**
+ * A pretty prose guide **in rhel7-guide.html** containing practical, actionable information for administrators
```

Comment:
File `rhel7-guide.html` don't exist, `guides/rhel7-guide-index.html` is the one with the nice Profile selector.